recipes/tl-expected/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.0.1":
+    url: https://github.com/yipdw/expected-1/archive/v1.0.1.zip
```

Comment:
Why don't you use TartanLlama/expected with a fixed commit instead? Using your fork, which is not the official, is not a good idea.

Reply:
I agree. I'll reverse this change and apply the strategy suggested in https://github.com/conan-io/conan-center-index/pull/556#discussion_r366203377.